doc/Integration.rst

```diff
@@ -0,0 +1,67 @@
+Integration
+===========
+
+Doctrine integration
+--------------------
+
+Independently from your preferred framework's characteristics, it is also possible to use plain
+Doctrine with the Money VO, using its Embeddables_ feature.
+
+.. _Embeddables: http://doctrine-orm.readthedocs.org/projects/doctrine-orm/en/latest/tutorials/embeddables.html
+
+Since you should not use annotations on the third party's library files, you'll want to use the `YAML mapping`_.
+
+.. _YAML mapping: http://doctrine-orm.readthedocs.org/projects/doctrine-orm/en/latest/reference/yaml-mapping.html#yaml-mapping
+
+The following two yaml files for Money and Currency turned out working well:
+
+.. code-block:: yaml
+
+    Money\Money:
+      type: embeddable
+      table: money # the persistence table name you'd like to use
+
+      embedded:
+        'currency':
+          class: '\Money\Currency'
```

Comment:
I wonder if the quotes are necessary. Sometimes you use them, sometimes not. Also, in this particular case: the slash in the class name is inconsistent with the rest of the docs.